### PR TITLE
vet change

### PR DESCRIPTION
**Pull request by shalkya**
make vetting relevant late game

### CODE REVIEW

**Comment by KionX** [comment]
Approve.

File: units/XSL0001/XSL0001_unit.bp [hunk 1]
```diff
@@ -125,13 +125,14 @@ UnitBlueprint {
...
-            Level2 = 9,
-            Level3 = 21,
-            Level4 = 36,
-            Level5 = 60,
+            Level1 = 6,
+            Level2 = 18,
+            Level3 = 42,
+            Level4 = 72,
+            Level5 = 120,
         },
```
Comment (ThrashAbaddon) [inline units/XSL0001/XSL0001_unit.bp:136]: You have the same `Regen` on couple of places. Can it be reused maybe?

**Comment by EldoreiJK** [comment]
Is this such a good idea? Commander t1 rush would be more rewarding, the same goes for all kind of telesnipes and all kinds of aggressive acu gameplay IMO.

**Comment by shalkya** [comment]
was decided to give 10 level of vet to ACU
As i'm not going to work on that anymore, i'm closing it

**Pull request closed without merging**